experimental/generation/generator/packages/library/templates/standard/PropertyToChange-add-PROPERTYName.dialog.lg

```diff
@@ -1,22 +1,22 @@
 [generator.lg](../generator.lg)
 
 # filename
-- ${libraryFilename(`PropertyToChange-add-PROPERTYName.dialog`)}
-
+- ${libraryFilename('PropertyToChange-add-PROPERTYName.dialog')}
 
 # template
 - ```
 {
     "$schema": "${appSchema}",
     "$kind": "Microsoft.OnAssignEntity",
-    "operation": "Set(PropertyToChange)",
-    "entity": "PROPERTYName",
+    "operation": "Add()",
```

Comment:
should this be Set() ?

Reply:
Default operation is Add() so this is correct.

---
In reply to: [478229881](https://github.com/microsoft/BotBuilder-Samples/pull/2690#discussion_r478229881) [](ancestors = 478229881)